storage/remote/read_handler.go

```diff
@@ -20,6 +20,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/prometheus/client_golang/prometheus/promauto"
```

Comment:
this can move down to near the other `client_golang` import

Reply:
Done.